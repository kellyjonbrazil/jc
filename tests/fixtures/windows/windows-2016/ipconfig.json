{
  "host_name": "WIN-SERVER16",
  "primary_dns_suffix": "somecompany.corp",
  "node_type": "Hybrid",
  "ip_routing_enabled": false,
  "wins_proxy_enabled": false,
  "dns_suffix_search_list": [
    "somecompany.corp"
  ],
  "adapters": [
    {
      "name_long": "Ethernet adapter Ethernet0",
      "name": "Ethernet0",
      "type": "Ethernet",
      "connection_specific_dns_suffix": null,
      "connection_specific_dns_suffix_search_list": [],
      "description": "Intel(R) 82574L Gigabit Network Connection",
      "physical_address": "00-0C-29-D6-12-2E",
      "dhcp_enabled": false,
      "autoconfiguration_enabled": true,
      "ipv6_addresses": [],
      "temporary_ipv6_addresses": [],
      "link_local_ipv6_addresses": [
        {
          "address": "fe80::40c:2f5f:6658:9ec3",
          "prefix_length": 2,
          "status": "Preferred"
        }
      ],
      "ipv4_addresses": [
        {
          "address": "192.168.22.153",
          "subnet_mask": "255.255.255.0",
          "status": "Preferred",
          "autoconfigured": false
        }
      ],
      "default_gateways": [
        "192.168.22.1"
      ],
      "dhcp_server": null,
      "dhcpv6_iaid": "50334761",
      "dhcpv6_client_duid": "00-01-00-01-29-4D-1C-4C-00-0C-29-A3-37-4E",
      "dns_servers": [
        "192.168.22.151"
      ],
      "primary_wins_server": null,
      "lease_expires": null,
      "lease_obtained": null,
      "netbios_over_tcpip": true,
      "media_state": null,
      "extras": []
    },
    {
      "name_long": "Tunnel adapter isatap.{17A4D79E-85D7-47FC-8972-ABBD8B58DC71}",
      "name": "isatap.{17A4D79E-85D7-47FC-8972-ABBD8B58DC71}",
      "type": "Tunnel",
      "connection_specific_dns_suffix": null,
      "connection_specific_dns_suffix_search_list": [],
      "description": "Microsoft ISATAP Adapter #2",
      "physical_address": "00-00-00-00-00-00-00-E0",
      "dhcp_enabled": false,
      "autoconfiguration_enabled": true,
      "ipv6_addresses": [],
      "temporary_ipv6_addresses": [],
      "link_local_ipv6_addresses": [],
      "ipv4_addresses": [],
      "default_gateways": [],
      "dhcp_server": null,
      "dhcpv6_iaid": null,
      "dhcpv6_client_duid": null,
      "dns_servers": [],
      "primary_wins_server": null,
      "lease_expires": null,
      "lease_obtained": null,
      "netbios_over_tcpip": null,
      "media_state": "Media disconnected",
      "extras": []
    }
  ],
  "extras": []
}
