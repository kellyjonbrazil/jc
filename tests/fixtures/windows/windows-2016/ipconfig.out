
Windows IP Configuration

   Host Name . . . . . . . . . . . . : WIN-SERVER16
   Primary Dns Suffix  . . . . . . . : somecompany.corp
   Node Type . . . . . . . . . . . . : Hybrid
   IP Routing Enabled. . . . . . . . : No
   WINS Proxy Enabled. . . . . . . . : No
   DNS Suffix Search List. . . . . . : somecompany.corp

Ethernet adapter Ethernet0:

   Connection-specific DNS Suffix  . :
   Description . . . . . . . . . . . : Intel(R) 82574L Gigabit Network Connection
   Physical Address. . . . . . . . . : 00-0C-29-D6-12-2E
   DHCP Enabled. . . . . . . . . . . : No
   Autoconfiguration Enabled . . . . : Yes
   Link-local IPv6 Address . . . . . : fe80::40c:2f5f:6658:9ec3%2(Preferred)
   IPv4 Address. . . . . . . . . . . : 192.168.22.153(Preferred)
   Subnet Mask . . . . . . . . . . . : 255.255.255.0
   Default Gateway . . . . . . . . . : 192.168.22.1
   DHCPv6 IAID . . . . . . . . . . . : 50334761
   DHCPv6 Client DUID. . . . . . . . : 00-01-00-01-29-4D-1C-4C-00-0C-29-A3-37-4E
   DNS Servers . . . . . . . . . . . : 192.168.22.151
   NetBIOS over Tcpip. . . . . . . . : Enabled

Tunnel adapter isatap.{17A4D79E-85D7-47FC-8972-ABBD8B58DC71}:

   Media State . . . . . . . . . . . : Media disconnected
   Connection-specific DNS Suffix  . :
   Description . . . . . . . . . . . : Microsoft ISATAP Adapter #2
   Physical Address. . . . . . . . . : 00-00-00-00-00-00-00-E0
   DHCP Enabled. . . . . . . . . . . : No
   Autoconfiguration Enabled . . . . : Yes
